src/web/client/test/integration/portalSchemaReader.test.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import * as schemaHelperUtil from "../../utilities/schemaHelperUtil";
+import * as portalSchemaReadeMock from "../integration/portalSchemaReader.mock";
+import { schemaEntityKey } from "../../../client/schema/constants";
+import Sinon, { stub } from "sinon";
+import {
+    getEntitiesSchemaMap,
+    getDataSourcePropertiesMap,
+    getEntitiesFolderNameMap,
+} from "../../../client/schema/portalSchemaReader";
+import { expect } from "chai";
+
+describe("portalSchemaReader", () => {
+    afterEach(() => {
+        Sinon.restore();
+    });
+    it("getEntitiesSchemaMap", () => {
```

Comment:
proper naming of the test case is necessary. Suppose you are testing it whether it is returning map DS for the schema or not then name it accordingly.
getEntitiesSchemaMap_withValidSchema_shouldBeMap

Reply:
Done